CMakePresets.json

```diff
@@ -1,16 +1,10 @@
 {
-  "version": 2,
-  "cmakeMinimumRequired": {
-    "major": 3,
-    "minor": 20,
-    "patch": 0
-},
+  "version": 4,
 
 "configurePresets": [
 {
   "name": "ci",
   "binaryDir": "${sourceDir}/build",
-  "generator": "Ninja",
```

Comment:
Please stop removing Ninja as the ci generator. We need it for Geoclaw since it makes heavy use of f90 modules